specification/validated-relationships-service-api.yaml

```diff
@@ -280,6 +281,81 @@ paths:
                 internalServerError:
                   $ref: '#/components/examples/InternalServerError'
 
+  /Consent:
+    get:
+        summary: Get proxy relationships
+        description: |
+          ## Overview
+          Use this endpoint to get the details about proxy relationships, including current status, based on a provided proxy's NHS Number.
```

Comment:
This description is a bit unclear why I would use this endpoint rather than GET /RelatedPerson. Unless I wanted the consent status. Is that the only publicly distinguishable difference?

Reply:
It's not overly clear I agree...will see about better use of language. /RelatedPerson is candidate relationships and /Consent is for retrieving proxy relationships - at various stages in their lifecycle,